### DISCUSSION TITLE
更新了官方数据跑不起来

### DISCUSSION THREAD

**Discussion by wsiqq23** [ssbuild/chatglm_finetuning#240] [General]
跑train.py的时候报size mismatch

**Comment by wsiqq23** [comment]
下载了最新的chatglm-6b的数据，pytorch_model_00001~00008.bin之后，再运行train.py就报类型错误了

**Reply by ssbuild** [reply]
> 下载了最新的chatglm-6b的数据，pytorch_model_00001~00008.bin之后，再运行train.py就报类型错误了

拉取一下dev 试试

**Reply by wsiqq23** [reply]
您好，现在能跑起来了，我用的4090的显卡，24G显存，运行data_utils.py和infer.py都没有问题，运行train.py的时候报出了两个错：
torch.cuda.OutOfMemory，以及DefaultCPUAllocator:not enough memory。
我已经将max_seq_length改成了256，仍然不行

**Reply by ssbuild** [reply]
> 您好，现在能跑起来了，我用的4090的显卡，24G显存，运行data_utils.py和infer.py都没有问题，运行train.py的时候报出了两个错： torch.cuda.OutOfMemory，以及DefaultCPUAllocator:not enough memory。 我已经将max_seq_length改成了256，仍然不行

修改max_seq_length ， 删除outout 下面的record 缓存， 重新制作才能生效，  最大批次改成1 应该可以的吧。

**Reply by wsiqq23** [reply]
是在chatglm_sft_config.py里面改的吧？train_batch_size:1,max_seq_length:256，重新运行data_utils.py，再运行train.py，仍然内存不足

**Reply by ssbuild** [reply]
> 是在chatglm_sft_config.py里面改的吧？train_batch_size:1,max_seq_length:256，重新运行data_utils.py，再运行train.py，仍然内存不足

不删除outout 下面的record 缓存， 不生效！！！！

**Reply by wsiqq23** [reply]
我是把整个output文件夹都删掉整的

**Reply by ssbuild** [reply]
内存不足 ，那就没办法了 。

**Reply by wsiqq23** [reply]
4090的显卡都跑不了吗……
